jenkins/integ-test-notification.jenkinsfile

```diff
@@ -24,13 +24,15 @@ pipeline {
     }
     options {
         timeout(time: 1, unit: 'HOURS')
-        buildDiscarder(logRotator(daysToKeepStr: '90'))
+        buildDiscarder(logRotator(daysToKeepStr: '30'))
```

Comment:
We need this to be 90 days or it is very hard to reference previous run in a new release.
Adding @prudhvigodithi as he made the initial change from 30 to 90.

Thanks.

Reply:
This is only for integration test notification. Do we still need it to be 90?

Reply:
I see. Feel like we might still need it to be consistent with integTest itself?
Adding @prudhvigodithi .

Thanks.

Reply:
Since today we are indexing the data to metrics cluster and also have the log in s3, we dont need 90 days.